apps/vr-tests/package.json

```diff
@@ -19,6 +19,7 @@
     "@fluentui/example-data": "^8.0.0-beta.0",
     "@fluentui/font-icons-mdl2": "^8.0.0-beta.12",
     "@fluentui/react": "^8.0.0-beta.42",
+    "@fluentui/react-avatar": "^0.8.2-16",
```

Comment:
what's the reason to have avatar versioned this way?

Reply:
I don't know why react-avatar is versioned that way. It's not something I changed. Maybe @ecraig12345 knows?

Reply:
Weird, I have no idea. But I see that some other early converged packages such as react-text have similar versions. @xugao do you have any thoughts on what the version ought to be instead?

Reply:
this is [the associated beachball bump](https://github.com/microsoft/fluentui/commit/0eac5198b9c5a921a0437de96305c4b28cc6048c). i am very puzzled..

Reply:
Hmm so I guess beachball has a default prerelease version of `-0`

![image](https://user-images.githubusercontent.com/5864305/105920301-fb1f0700-5feb-11eb-8186-9310edd8aaaf.png)

Reply:
If you tell it to bump prerelease on something that didn't previously have a prerelease